src/pages/ErrorPage/ErrorPage.jsx

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import PageLayout from '../../PageLayout/PageLayout';
-import styles from './ErrorPage.scss';
+import { DivError, ImageError } from './ErrorPage.Styles';
 import image from '../../../public/error.jpg';
 
 function ErrorPage() {
   return (
     <PageLayout>
-      <div className={styles.error}>
-        <img src={image} className={styles.error__img} alt="Error Pic" />
-      </div>
+      <DivError>
+        <ImageError src={image} alt="Error Pic" />
```

Comment:
ImageError means that it is an error of image, but in your case I suppose it should be about image that looks like and error thing. So more closely it woud be ErrorImage, maybe something else